store/file/square.go

```diff
@@ -141,3 +150,34 @@ func oppositeAxis(axis rsmt2d.Axis) rsmt2d.Axis {
 	}
 	return rsmt2d.Col
 }
+
+func newODSWriter(s square, size int) *odsWriter {
+	return &odsWriter{
+		ods:   s,
+		total: size * size,
+	}
+}
+
+type odsWriter struct {
+	ods square
+	// current is the amount of Shares stored in square that have been written by odsWriter. When
+	// current reaches total, odsWriter will prevent further reads by returning io.EOF
+	current, total int
+}
+
+func (w *odsWriter) WriteTo(writer io.Writer, limit int) (int, error) {
```

Comment:
`WriteTo` is a method from a well-known interface https://pkg.go.dev/io#WriterTo

Quite often breaking such unspoken rules might result in a future incompatibilities between implementation and stdlib interfaces. Just saying.